assets/scss/_header.scss

```diff
@@ -225,6 +225,9 @@
             }
         }
         > a {
+            @media (min-width: $min-desktop) {
```

Comment:
I think @media (hover: hover) is more appropriate here, it detects if hover state is shown, rather than basing it on device style. Will update.